mods/cnc/languages/rules/en.ftl

```diff
@@ -507,40 +570,53 @@ actor-bike =
     Can attack Aircraft.
       Strong vs Vehicles, Tanks
       Weak vs Infantry
+   .encyclopedia = The Nod bike is a very fast vehicle armed with armor-piercing rockets, which makes it great for hit and run and harvester harassment.
+     Be wary that its rockets are slow and have a hard time hitting fast-moving targets.
 
 actor-jeep =
    .name = Hum-Vee
    .description = Fast scout and anti-infantry vehicle.
       Strong vs Infantry
       Weak vs Tanks
+   .encyclopedia = Strong versus light vehicles and infantry, Hum-vees are great at scouting and eliminating isolated infantry.
+     They're excellent for dealing with Nod bike spam and should be built instead of tanks in such engagements. Although a little slower than the Nod buggy, it has more health.
 
 actor-ltnk =
    .name = Light Tank
    .description = Fast, light tank.
       Strong vs Vehicles, Tanks
       Weak vs Infantry
+   .encyclopedia = With its great speed and high health pool, the light tank works great as a frontline unit in various compositions despite its damage output.
+     It should be combined with bikes, stealth tanks, or infantry when facing medium tanks.
 
 actor-mtnk =
    .name = Medium Tank
    .description = General-Purpose GDI Tank.
       Strong vs Tanks, Vehicles
       Weak vs Infantry
+   .encyclopedia = The medium tank is the workhorse of GDI, offering a good balance of speed, durability and firepower. It will easily destroy Nod light tanks and can tear through structures if it sneaks into a base.
+     Mix minigunners with your medium tank armies to deal with enemy infantry.
 
 actor-htnk =
    .name = Mammoth Tank
    .description = Heavily armored GDI Tank.
     Can attack Aircraft.
       Strong vs Everything
+   .encyclopedia = The Mammoth Tank is a crawling battle station, capable of self-repair and engaging any threat. It is particularly effective against enemy armor, but can also destroy small amounts of infantry or aircraft.
+     When in an enemy base, put it on aggressive stance to have it carve a path of destruction as it moves.
 
 actor-msam =
    .name = Rocket Launcher
    .description = Long range rocket artillery.
       Strong vs all Ground units.
+   .encyclopedia = Referred to as  “MLRS” by players, this artillery platform fires volleys of rockets at distant targets. It is particularly effective against light vehicles and does moderate damage vs infantry, buildings, and heavy armor.
+     A strong unit to build vs enemy artillery, its high bullet velocity and damage to light vehicles makes it the perfect counter to Nod artillery.
 
 actor-mlrs =
    .name = Mobile S.A.M.
    .description = Powerful anti-air unit.
     Cannot attack Ground units.
+   .encyclopedia = Generally referred to as “MSAM”, this Nod vehicle is a dedicated anti-air vehicle when their bikes do not cut it. It fires slow-moving missiles with powerful splash damage.
```

Comment:
If you mean `MSAM` as in the game rules, then this is pointing on the wrong unit. Maybe leave it out.